app/src/main/res/layout/home_fragment.xml

```diff
@@ -20,24 +21,24 @@
             android:layout_marginTop="30dp"
             android:background="@drawable/frame"
             android:backgroundTint="#C6B2EA"
+            android:onClick="@{() -> viewModel.onD20Clicked()}"
             android:src="@drawable/d20_svg"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            android:onClick="@{() -> viewModel.onD20Clicked()}"/>
```

Comment:
onD20Clicked will make little sense to most collegues as a name? what is D20?  what is a d20_button? 

clear names